econml/dml/dml.py

```diff
@@ -716,13 +741,19 @@ class LinearDML(StatsModelsCateEstimatorMixin, DML):
 
     def __init__(self, *,
                  model_y='auto', model_t='auto',
```

Comment:
We should probably update our docstrings to say that users can pass something like 'forest' to model_y right?

Reply:
Yes, I think we should probably create a whole new documentation section on how to specify model selectors, because there are a number of possibilities now.  But I was planning on deferring that until after this PR.